core/request.py

```diff
@@ -298,16 +299,23 @@ def processRequest(self) -> None:
                     logging.exception(newE)
                     res = None
             if not res:
-                with (conf["viur.instance.core_base_path"]
-                      .joinpath(conf["viur.errorTemplate"])
-                      .open() as tpl_file):
-                    tpl = Template(tpl_file.read())
-                res = tpl.safe_substitute({
-                    "error_code": e.status,
-                    "error_name": translate(e.name),
-                    "error_descr": e.descr,
-                })
-                extendCsp({"style-src": ['sha256-Lwf7c88gJwuw6L6p6ILPSs/+Ui7zCk8VaIvp8wLhQ4A=']})
+                if (len(self.pathlist) > 0 and any(x in self.pathlist[0] for x in ["vi", "json"])) or \
+                        utils.currentRequest.get().response.headers["Content-Type"] == "application/json":
+                    utils.currentRequest.get().response.headers["Content-Type"] = "application/json"
+                    res = json.dumps({
+                        "error_code": e.status,
+                        "error_name": str(translate(e.name)),
+                        "error_descr": e.descr,
+                    })
```

Comment:
Hi! I don't like this redundant `error_`-prefix, as a returned HTTP-error is by definition an error.
I'm also of the opinion as @sveneberth stated in #549, that we shouldn't translate the name in this case. 

What about this result, with both the translated and original name, and an improved set of keys?
```suggestion
                    res = json.dumps({
                        "status": e.status,
                        "reason": e.name,
                        "descr": str(translate(e.name)),
                        "hint": e.descr
                    })
```

- **status** is the HTTP status code.
- **reason** is the unmodified HTTP reason phrase.
- **descr** is the possibly translated reason phrase.
- **hint** is a hint about what went wrong. It can be null in some cases.